lib/Kaltura/AllInOneVideoPackPlugin.php

```diff
@@ -104,12 +105,15 @@ public function adminEnqueueScripts() {
 		), KALTURA_PLUGIN_VERSION, true );
 		wp_register_style( 'kaltura-jquery.fileupload-ui', KalturaHelpers::cssUrl( 'chunked-file-upload-jquery/css/jquery.fileupload-ui.css' ), array( ), KALTURA_PLUGIN_VERSION );
 		wp_register_style( 'kaltura-jquery.fileupload-ui-kaltura', KalturaHelpers::cssUrl( 'chunked-file-upload-jquery/css/jquery.fileupload-ui-kaltura.css' ), array( 'kaltura-jquery.fileupload-ui', 'kaltura-bootstrap' ), KALTURA_PLUGIN_VERSION );
-
-		wp_enqueue_script( 'kaltura', KalturaHelpers::jsUrl( 'js/kaltura.js' ), array(), KALTURA_PLUGIN_VERSION, false );
-		wp_enqueue_script( 'kaltura-admin', KalturaHelpers::jsUrl( 'js/kaltura-admin.js' ), array(), KALTURA_PLUGIN_VERSION, false );
-
-		wp_enqueue_style( 'kaltura-admin', KalturaHelpers::cssUrl( 'css/admin.css' ), array(), KALTURA_PLUGIN_VERSION );
-		wp_enqueue_style( 'kaltura' );
+		wp_register_style( 'kaltura-admin', KalturaHelpers::cssUrl( 'css/admin.css' ), array(), KALTURA_PLUGIN_VERSION );
+		
+		// If admin page is post.
+		$screen = get_current_screen();
+		if ( 'post' === $screen->base ) {
```

Comment:
I'm not sure if it is a good idea to make such conditions here, it would be better to include the scripts in the relevant actions/controllers